tools/scrape_abstracts.py

```diff
@@ -101,4 +102,4 @@ def loop_until_end(text, start, end, sep=''):
 
 # dump to json
 with open('abstract.json', 'w') as dest:
-    json.dump(abstracts, dest, indent=1, ensure_ascii=True)
+    json.dump(abstracts, dest, ensure_ascii=False)
```

Comment:
this is unrelated to authors fix...  I would just keep it all indented so diff is manageable to inspect please.

Reply:
The initial (i.e., current) `abstract.json` wasn't indented because of an error on my part—that is, it's only one line. I just edited the script to match what I had accidentally uploaded initially, but I'll revert it and add the correctly-indented JSON if you'd like.